db/companyinfo.txt

```diff
@@ -1,102 +0,0 @@
-const data = [
```

Comment:
This is a `.txt` file but you're writing JS in it for some reason, I'd make sure to remove this...